Frontend/CO.CDP.OrganisationApp/appsettings.AwsIntegration.json

```diff
@@ -0,0 +1,3 @@
+{
+  "GoogleTagId": "G-2TLMW60D58"
```

Comment:
is it correct that we use same GoogleTagId for 3 different envs?

Reply:
That is what business provided to us